packages/plugin-commands-audit/test/doctor.ts

```diff
@@ -0,0 +1,35 @@
+import { doctor } from '@pnpm/plugin-commands-audit'
+import sinon from 'sinon'
+import { logger } from '@pnpm/logger'
+
+test('doctor --config', async () => {
+  const oldExecPath = process.execPath
+  const oldPlatform = process.platform
+  const oldEnv = process.env
+  const HOMEBREW_PREFIX = '.'
+
+  process.env = { ...oldEnv, HOMEBREW_PREFIX }
+  process.execPath = HOMEBREW_PREFIX
+  // platform is read only
+  Object.defineProperty(process, 'platform', {
+    value: 'darwin',
+  })
+
+  const reporter = sinon.spy(logger, 'warn')
```

Comment:
For test cases, I don't know why the anonymous function `sinon.spy()` doesn't work and I need to add params to it. Is there any configs about it I missed?

Reply:
Don't use sinon. Use jest.fn or other jest functions for mocking.

Reply:
ok